.github/workflows/e2e_tests.yml

```diff
@@ -126,7 +115,13 @@ jobs:
 
     strategy:
```

Comment:
For now add needs on eks-test and aks-test for gke. And add a note and a TODO why this is needed. And move the gke job to below AKS